src/core/AutoRest.Core/Validation/ValidationExceptionName.cs

```diff
@@ -1,21 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-namespace AutoRest.Core.Validation
-{
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Xms")]
```

Comment:
I eliminated this because it required Validators to pre-register in this file, which would make it more of a pain for generators to add their own.